doc/user_guide/examples.rst

```diff
@@ -51,6 +51,12 @@ to provide an overview of the various examples so that a user can find
 one that is appropriate to them. For details of how to run each
 example please see the ``README.md`` files in the associated directories.
 
+The ``check_examples`` bash script (in the ``examples`` directory) is
```

Comment:
This should now be part of our review rules i.e. that check_examples -c should be manually done and should be successful (until it is added as part of travis). Could you please add.

Reply:
Good point - done.